### PR TITLE
PIM-11249: Keep resolution metadata (DPI) of transformed asset images

### PR DESCRIPTION
**Pull request by LevFlavien**
**Description (for Contributor and Core Developer)**

https://akeneo.atlassian.net/browse/PIM-11249

EE PR: https://github.com/akeneo/pim-enterprise-dev/pull/20356

**Definition Of Done (for Core Developer only)**

- [x] Tests
- [x] Changelog (maintenance bug fixes)

### CODE REVIEW

File: tests/back/Platform/Specification/Bundle/UIBundle/Imagine/IccStripFilterIntegration.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+use PHPUnit\Framework\TestCase;
+
+class IccStripFilterIntegration extends TestCase
+{
+    /**
+     * @dataProvider fileNameProvider
+     */
+    public function testResolutionMetadataIsKept(string $fileName): void
+    {
+        $sut = new IccStripFilter();
```
Comment (olivierpontier) [inline tests/back/Platform/Specification/Bundle/UIBundle/Imagine/IccStripFilterIntegration.php:17]: what does `sut` mean?

**Review by olivierpontier — APPROVED** [review]

File: tests/back/Platform/Specification/Bundle/UIBundle/Imagine/IccStripFilterIntegration.php (hunk 1 above)
Comment (mmetayer) [reply tests/back/Platform/Specification/Bundle/UIBundle/Imagine/IccStripFilterIntegration.php:17]: @olivierpontier it means "system under test"

**Review by antoine-bourgault — APPROVED** [review]

File: tests/back/Platform/Specification/Bundle/UIBundle/Imagine/IccStripFilterIntegration.php (hunk 1 above)
Comment (LevFlavien) [reply tests/back/Platform/Specification/Bundle/UIBundle/Imagine/IccStripFilterIntegration.php:17]: system under test

File: tests/back/Platform/Specification/Bundle/UIBundle/Imagine/IccStripFilterIntegration.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Specification\Akeneo\Platform\Bundle\UIBundle\Imagine;
```
Comment (mmetayer) [inline tests/back/Platform/Specification/Bundle/UIBundle/Imagine/IccStripFilterIntegration.php:3]: wrong namespace, I'd put that test somewhere in the `AkeneoTest\Platform\Integration\UI` namespace

Comment (LevFlavien) [reply tests/back/Platform/Specification/Bundle/UIBundle/Imagine/IccStripFilterIntegration.php:3]: Done here https://github.com/akeneo/pim-community-dev/pull/20368

**Pull request merged**